test/fixtures/fieldset.html

```diff
@@ -60,6 +60,11 @@
     <input id="fieldset-disabled-fieldset-enabled-input" type="text" value="Input not tabbable/focusable" />
   </fieldset>
 
+  <fieldset id="fieldset-disabled-fieldset-disabled" disabled>
+    <legend><button id="fieldset-disabled-fieldset-disabled-legend-button">Button not tabbable/focusable</button></legend>
+    <input id="fieldset-disabled-fieldset-disabled-input" type="text" value="Input not tabbable/focusable" />
+  </fieldset>
```

Comment:
Thanks for adding this! The old version of the code doesn't catch this one. 🎉 

Adding this covers the following tests:
- https://github.com/focus-trap/tabbable/blob/master/test/e2e/tabbable.e2e.js#L259
- https://github.com/focus-trap/tabbable/blob/master/test/e2e/focusable.e2e.js#L230

But it doesn't cover these two:
- https://github.com/focus-trap/tabbable/blob/master/test/e2e/isFocusable.e2e.js#L302
- https://github.com/focus-trap/tabbable/blob/master/test/e2e/isTabbable.e2e.js#L293

For some reason, those last 2 test files don't use the fixtures. 🤷‍♂️ 

Can you add to those so that `isFocusable()` and `isTabbable()` are tested for this additional case you uncovered that we weren't handling properly?